src/lib/flight_tasks/FlightTasks.hpp

```diff
@@ -81,6 +81,12 @@ class FlightTasks
 	 */
 	const vehicle_local_position_setpoint_s getPositionSetpoint();
 
+	/**
+	 * Get the reset counters from the current task
```

Comment:
The name says that but I would probably ask myself what reset counters.
```suggestion
	 * Get the estimation reference frame reset counters from the current task
```
?